src/components/auth/Auth.js

```diff
@@ -3,14 +3,20 @@ import React, { Component } from 'react';
 
 import Login from './Login';
 
+
```

Comment:
Re-enable eslint on this file when you get a chance

Reply:
refactored from class to functional component - done!